memcached-api.php

```diff
@@ -116,6 +118,8 @@ class Memcached {
 
 	const HAVE_SASL;
 
+	const HAVE_TLS;
```

Comment:
Do you need this when you have `OPT_USE_TLS` option above?

Reply:
yes, HAVE_TLS indicates if the package was built with TLS support (same as for HAVE_SASL), and OPT_USE_TLS is an option that the user can set in order to use TLS in a specific client object

Reply:
OK, it is fine to have a compiler flag to compile out the SSL code so you can build the client library without OpenSSL.